src/components/page/blocks/ContentBlock.jsx

```diff
@@ -24,12 +25,26 @@ class ContentBlock extends React.PureComponent {
   render() {
     const { block, row } = this.props;
     const { content } = block || {};
+    const transComponents = {
```

Comment:
mapping for basic HTML elements that need to be parsed. As we load in more content from other investigations this may grow. This is mostly so that we can support nested content which i18next will not parse using just the config-supported HTML elements.